docker/README.md

```diff
@@ -41,5 +41,7 @@ ngrok http 8082
 
 You will see it give a forwarding address like this one:
  http://e0747cffd8a3.ngrok.io
+ 
+You may need to set your `siteurl` and `home` `wp_option`s to the new url. You can do this with phpMyAdmin or WP-CLI.
```

Comment:
Seems like this may only happen in some circumstances. LTGM.
I think we may need to call out that this should only be a temporary change while setting up WCPay.

```suggestion
You may need to temporarily set your `siteurl` and `home` `wp_option`s to the new url. You can do this with phpMyAdmin or WP-CLI.
```

Reply:
Good tip about "temporary", as I understand it the ngrok tunnel is only needed for initial onboarding connection.